################################################################################
SHORT DESCRIPTION: 
################################################################################
Fix Windows XP standby with cirrus VGA

################################################################################
LONG DESCRIPTION: 
################################################################################
Fix it by telling OSPM don't power down vga card on entering S3
state. The trick works for XP and Windows2003, but Vista still refuse
to allow S3.

################################################################################
CHANGELOG 
################################################################################

################################################################################
REMOVAL 
################################################################################

################################################################################
UPSTREAM PLAN
################################################################################

################################################################################
INTERNAL DEPENDENCIES 
################################################################################

################################################################################
PATCHES 
################################################################################
--- a/tools/libacpi/dsdt.asl
+++ b/tools/libacpi/dsdt.asl
@@ -53,25 +53,6 @@ DefinitionBlock ("DSDT.aml", "DSDT", 2,
            Name (_ADR, 0x00)
            Name (_BBN, 0x00)
 
-           /* Make cirrues VGA S3 suspend/resume work in Windows XP/2003 */
-           Device (VGA)
-           {
-               Name (_ADR, 0x00020000)
-
-               Method (_S1D, 0, NotSerialized)
-               {
-                   Return (0x00)
-               }
-               Method (_S2D, 0, NotSerialized)
-               {
-                   Return (0x00)
-               }
-               Method (_S3D, 0, NotSerialized)
-               {
-                   Return (0x00)
-               }
-           }
-
            Method (_CRS, 0, NotSerialized)
            {
                Store (ResourceTemplate ()
